user_docs/en/userGuide.t2t

```diff
@@ -3018,7 +3018,7 @@ You can also enable or disable multiple add-ons at once by selecting multiple ad
 +++ Reviewing add-ons and reading reviews +++[AddonStoreReviews]
 Before installing an add-on, you may want to read reviews by others.
 Also, it may be helpful to other users to provide feedback about add-ons you have tried.

```

Comment:
Perhaps better wording here, since "may" is too weak a word IMO.

```suggestion
Also, it is helpful to other users to provide feedback about add-ons you have tried.
```